src/cow-react/modules/limitOrders/utils/getOrderFilledAmount.ts

```diff
@@ -0,0 +1,34 @@
+import BigNumber from 'bignumber.js'
+import { Order } from 'state/orders/actions'
+import { ZERO_BIG_NUMBER } from '@cowprotocol/cow-js'
+
+/**
+ * Get order filled amount, both as raw amount (in atoms) and as percentage (from 0 to 1)
+ *
+ * @param order The order
+ */
+
+interface FilledAmountResult {
+  amount: BigNumber
+  percentage: BigNumber
+}
+
+export function getOrderFilledAmount(order: Order): FilledAmountResult {
+  let executedAmount, totalAmount
```

Comment:
I wonder if these functions are a direct replication of what we do in the explorer. 

Did you check there the implementation.

Would it make sense to make these things in the SDK, or this adds too much overhead?